suite-native/transaction-management/src/components/fees/CustomFee/CustomFeeBottomSheet.tsx

```diff
@@ -47,9 +54,16 @@ export const CustomFeeBottomSheet = ({
 
     const handleSetCustomFee = handleSubmit(() => {
         setValue('feeLevel', 'custom');
-        const feePerUnit = getValues('customFeePerUnit');
-        const feeLimit = getValues('customFeeLimit');
-        onCustomFeeSet(feePerUnit, feeLimit);
+        const customFeePerUnit = getValues(FEE_PER_UNIT_FIELD_NAME);
+        const customFeeLimit = getValues(FEE_LIMIT_FIELD_NAME);
+        const customMaxFeePerGas = getValues(MAX_FEE_PER_GAS_FIELD_NAME);
+        const customMaxPriorityFeePerGas = getValues(MAX_PRIORITY_FEE_PER_GAS_FIELD_NAME);
```

Comment:
Nitpick:
```suggestion
        const { customFeePerUnit, customFeeLimit, customMaxFeePerGas, customMaxPriorityFeePerGas } = getValues();
```

Although I understand why using global constants can be beneficial.